src/snowflake/cli/api/rest_api.py

```diff
@@ -124,8 +126,14 @@ def determine_url_for_create_query(self, object_type: str) -> str:
         """
         plural_object_type = _pluralize_object_type(object_type)
 
+        query_params = ""
+
+        if replace or if_not_exists:
+            param = "ifNotExists" if if_not_exists else "orReplace"
+            query_params = f"?createMode={param}"
```

Comment:
Please consider building params using `urlencode`
https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode

Reply:
I thought about it, but since whole url is build using f-strings, i went that way.

Reply:
Done- in the query params part. Rest of the url building will be changed in separate PR, as it is out of scope for this one.